### PR TITLE
`berkeley`: Update API GUI how-to guide related to `/planned_processes` endpoints

### PR DESCRIPTION
**Pull request by brynnz22**
This PR updates the API documentation to work with the new planned process endpoints. I was not able to test the endpoints while I worked because I think perlmutter is down today. I will test tomorrow, but I updated the docs assuming they work as expected. 

This PR addresses issue: NMDC_documentation

I also updated the links to what they will be once Berkeley is merged in, so some of them technically won't work if you try them right now. Not sure if we want to wait to merge this PR until Berkeley is merged in. For example this link: https://microbiomedata.github.io/nmdc-schema/MetabolomicsAnalysis/ was updated from https://microbiomedata.github.io/nmdc-schema/MetabolomicsAnalysisActivity/

### CODE REVIEW

**Comment by aclum** [comment]
We should wait to merge this in until berkeley is merged in and we know the release of nmdc-runtime. Otherwise this looks good to me.

**Review by aclum — APPROVED** [review]

File: docs/howto_guides/api_gui.md
Comment (eecavanna) [inline docs/howto_guides/api_gui.md:72]: I suggest including the curly braces as a way of distinguishing the constant parts of the path from the variable parts of the path:

```suggestion
If the planned process identifier is known, the metadata can be retrieved using the `GET /planned_processes/{planned_process_id}` endpoint. Note that only one metadata record for a planned process may be returned at a time using this method.<br/>
```

**Comment by eecavanna** [comment]
Looks good to me—thank you, @brynnz22!

I left one comment about the formatting of a URL.

I agree with @aclum about holding off on merging this until we have switched our production systems over to using the Berkeley schema.

**Comment by eecavanna** [comment]
I'll merge this into `main` and deploy it to the production documentation website now.

**Pull request merged**
